rasa/shared/core/training_data/story_writer/yaml_story_writer.py

```diff
@@ -102,6 +104,7 @@ def stories_to_yaml(
 
         Args:
             story_steps: Original story steps to be converted to the YAML.
+            is_test_story: `True` if the story is an end-to-end conversation test story.
```

Comment:
I think that's confusing: it's either test story, or end-to-end story.
I understand that they share the same syntax, but then the name should be adapted imo.

Reply:
but there is test story (containing only core syntax) and there is e2e test story which contain data to test nlu as well

Reply:
I chose `end-to-end conversation test story` to make it very explicit that it's a test containing full user messages in contrast to a regular Core tests file. If we agree that `conversation tests` mean that it will always be containing real conversation data, then I'm happy to rename 👍🏻